src/commcare_cloud/ansible/roles/commcarehq/tasks/celery_cron.yml

```diff
@@ -1,27 +1,59 @@
-# To remove celery tasks that are running for more than 36 hours.
-#
+# To remove old celery tasks that are running from long time
+# SoftKill: SIGTERM, HardKill: SIGKILL
+
 - name: Celery cron jobs for soft kill tasks running from last 36h
   become: true
   cron:
     name: "Sending SIGTERM to celery processes running morethan 36h"
-    job: "kill -15 $(ps -eo comm,pid,etimes | awk '/celeryd:/ {if ($4 > 36*60*60) { print $3 }}')"
+    cron_file: celery_sigterm
+    state: absent
+  tags:
+    - cron
+
+- name: Cron job to Kill Celery tasks running from last 46h
+  become: true
+  cron:
+    name: "Sending SIGTERM to celery processes running morethan 46h"
+    cron_file: celery_sigkill
+    state: absent
+  tags:
+    - cron
+
+- name: Copy celery cron script
+  become: yes
+  template:
+    src: "celery_task_kill.sh.j2"
```

Comment:
For some reason this line is giving me

```
fatal: [10.202.12.80]: FAILED! => {"changed": false, "msg": "Could not find or access 'celery_task_kill.sh.j2'\nSearched in:\n\t/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/roles/commcarehq/tasks/templates/celery_task_kill.sh.j2\n\t/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/roles/commcarehq/tasks/celery_task_kill.sh.j2\n\t/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/templates/celery_task_kill.sh.j2\n\t/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/celery_task_kill.sh.j2 on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option"}
```
Here's the text there better formatted:
```
Could not find or access 'celery_task_kill.sh.j2'
Searched in:
	/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/roles/commcarehq/tasks/templates/celery_task_kill.sh.j2
	/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/roles/commcarehq/tasks/celery_task_kill.sh.j2
	/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/templates/celery_task_kill.sh.j2
	/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/celery_task_kill.sh.j2 on the Ansible Controller.
If you are using a module and expect the file to exist on the remote, see the remote_src option
```
If I change this line to `src: "../templates/celery_task_kill.sh.j2"` instead, then it runs fine. Not sure if it is something wrong with my setup or with this line. I would expect it to work as it is here just fine, just like https://github.com/dimagi/commcare-cloud/blob/673a27383c7371da5012cd3a6fb76689b7176e6f/src/commcare_cloud/ansible/roles/nginx/tasks/site.yml#L24 does.

I ran this this way: `cchq production deploy-stack --limit celery --tags cron`

Reply:
@dannyroberts that's odd. It appears to have looked in the correct place as well: `/Users/danielroberts/dimagi/commcare-cloud/src/commcare_cloud/ansible/roles/commcarehq/tasks/templates/celery_task_kill.sh.j2`. Did that file not exist?

Reply:
Is this in `deploy_commcarehq.yml`? that would make total sense. Why are we even doing it like this at all and not just running the tasks as part of the main role exection?

Reply:
That's a good point. It's nice to have these related tasks be in their own file, but I agree it'd be cleaner to import these tasks from the `celery.yml` tasks file (also in the `commcarehq` role) which is already executed from `deploy_commcarehq.yml` in the same way

Reply:
Looks like we never got around to this. @prem-fissionhq, @sanjay2916 bumped into this today and may make similar changes locally and/or PR the fix, so just make sure you coordinate with him on it

Reply:
@dannyroberts I'm working on it. thanks

Reply:
@dannyroberts this worked for me too. 
"src: "../templates/celery_task_kill.sh.j2" without any issue, testing other ways too.

Reply:
While that does work, I think what I suggested above is the way that ansible would recommend. By using `import_role` a number of default paths are set up, including the template directory, letting you use the normal conventions.

Reply:
@dannyroberts updated the file as per your suggestions and tested in staging and it worked fine for us. 
please review, thanks
https://github.com/dimagi/commcare-cloud/pull/4329